test/CreateConfig.test.js

```diff
@@ -559,28 +559,6 @@ describe('createConfig', () => {
     expect(config).toMatchSnapshot();
   });
 
-  it('spdy option', () => {
-    const config = createConfig(
-      webpackConfig,
-      Object.assign({}, argv, { https: true, spdy: true }),
-      { port: 8080 }
-    );
-
-    expect(config).toMatchSnapshot();
-  });
-
-  it('spdy option (in devServer config)', () => {
-    const config = createConfig(
-      Object.assign({}, webpackConfig, {
-        devServer: { https: true, spdy: true },
-      }),
-      argv,
-      { port: 8080 }
-    );
-
-    expect(config).toMatchSnapshot();
-  });
-
   it('http2 option', () => {
```

Comment:
Don't remove this test, just move this in `Http2` test

Reply:
I think this would stay in `CreateConfig.test.js` since it is only confirming that the config option appears after passing through the config helper, right? `Http2` test should be for actual functionality of the server, not just confirming that the config options get through to the server

Reply:
:+1: